Builders/src/main/scala/Builders.scala

```diff
@@ -0,0 +1,266 @@
+package com.gu.nitf.model.builders
+
+import java.net.URI
+import java.time.{LocalDate, LocalDateTime}
+
+import com.gu.nitf.model._
+import com.gu.nitf.scalaxb._
+import com.gu.scalaxb.{dataRecord => dr}
+import com.gu.scalaxb._
+import scalaxb._
+
+import scala.xml.NodeSeq
+
+
+object `package` {
```

Comment:
isn't `package object builders` what you meant?

Reply:
I think your expression would create a package object for the package `com.gu.nitf.model.builders.builders`, which isn't what I'm after.

The current definition allows the classes in the package to access the members of this object without having to import them.

Reply:
aaah you're right 👍